datahub-web-react/src/app/search/SearchResultList.tsx

```diff
@@ -104,7 +109,7 @@ export const SearchResultList = ({
                                 style={{ fontSize: 18, color: ANTD_GRAY[8] }}
                                 description={`No results found for "${query}"`}
                             />
-                            <Button onClick={() => navigateToSearchUrl({ query: '*', page: 0, history })}>
+                            <Button onClick={onClickExploreAll}>
```

Comment:
Debated pulling the ExploreAll btn into its own component. There's 3 of them. All with slightly different visuals and different event types. If we consolidate the designs it would make more sense to have a reusable component.

Reply:
Agree with this. Thanks for refactoring it to be a bit easier to read!